ethernet.rflx

```diff
@@ -4,6 +4,53 @@ package Ethernet is
    type Type_Length is range 46 .. 2**16 - 1 with Size => 16;
    type TPID is range 16#8100# .. 16#8100# with Size => 16;
    type TCI is mod 2**16;
+   type Ether_Type is
+      (ET_IPv4               => 16#0800#,
+       ET_ARP                => 16#0806#,
+       ET_Wake_on_LAN        => 16#0842#,
+       ET_AVTB               => 16#22F0#,
+       ET_TRILL              => 16#22F3#,
+       ET_SRP                => 16#22EA#,
+       ET_RARP               => 16#8035#,
+       ET_VLAN_Tag           => 16#8100#,
+       ET_SLPP               => 16#8102#,
+       ET_VLACP              => 16#8103#,
+       ET_IPX                => 16#8137#,
+       ET_QNX_Qnet           => 16#8204#,
+       ET_IPv6               => 16#86DD#,
+       ET_Flow_Control       => 16#8808#,
+       ET_LACP               => 16#8809#,
+       ET_MPLS_Unicast       => 16#8847#,
+       ET_MPLS_Multicast     => 16#8848#,
+       ET_PPPoE_Discovery    => 16#8863#,
+       ET_PPPoE_Session      => 16#8864#,
+       ET_Homeplug_10_MME    => 16#887B#,
+       ET_EAP_over_LAN       => 16#888E#,
+       ET_PROFINET           => 16#8892#,
+       ET_ATA_over_Ethernet  => 16#88A2#,
+       ET_EtherCAT           => 16#88A4#,
+       ET_Ethernet_Powerlink => 16#88AB#,
+       ET_GOOSE              => 16#88B8#,
+       ET_GSE                => 16#88B9#,
+       ET_SV                 => 16#88BA#,
+       ET_LLDP               => 16#88CC#,
+       ET_SERCOS_III         => 16#88CD#,
+       ET_WSMP               => 16#88DC#,
+       ET_MRP                => 16#88E3#,
+       ET_MAC_Security       => 16#88E5#,
+       ET_PBB                => 16#88E7#,
+       ET_PTP                => 16#88F7#,
+       ET_NC_SI              => 16#88F8#,
+       ET_PRP                => 16#88FB#,
+       ET_CFM                => 16#8902#,
+       ET_FCoE               => 16#8906#,
+       ET_FCoE_Init          => 16#8914#,
+       ET_RoCE               => 16#8915#,
+       ET_TTE                => 16#891D#,
+       ET_HSR                => 16#892F#,
+       ET_CTP                => 16#9000#,
+       ET_VLAN_Tag_Double    => 16#9100#,
+       ET_Redundancy_Tag     => 16#F1C1#) with Size => 16, Always_Valid;
```

Comment:
Inconsistent formatting (see previous comment).

Reply:
Fixed in ace51fd.